### PR TITLE
more adjustments

### PR DESCRIPTION
**Pull request by briandooley**

### CODE REVIEW

**Comment by grdryn** [comment]
lgtm :+1:

File: modules/ROOT/pages/_partials/gs-accessing-prometheus-grafana-proc.adoc [hunk 1]
```diff
@@ -7,15 +7,17 @@ endif::[]
...
-. In the *My Projects* section, click *View All*.
+. On the *All Services* tab, click the *Open console* button for *Red Hat OpenShift*.
+. Click the *My Projects* link.
 . Click the *Managed Service Monitoring* project.
-. From the sidebar, select *Applications* -> *Routes*.
-. To analyze and visualize your platform metrics using Grafana, click the hostname for _grafana-route_.
-+
-To query your platform metrics using Prometheus, click the hostname for _prometheus-route_.
+. In the sidebar, select *Applications* -> *Routes*.
+. In the *Routes* pane, you can choose to query or visualize the cluster metrics by clicking the host-names for different routes:
```
Comment (bhardesty) [inline modules/ROOT/pages/_partials/gs-accessing-prometheus-grafana-proc.adoc]: Here and in the sub-steps, "host name" should be two words (no hyphen, according to IBM manual of style).

**Comment by briandooley** [comment]
Thanks, I've changed the references to "hostname"s now. @bhardesty Can you give it one more look to sign it off?

**Comment by bhardesty** [comment]
@briandooley lgtm

File: modules/ROOT/pages/_partials/gs-accessing-prometheus-grafana-proc.adoc [hunk 2]
```diff
@@ -7,15 +7,17 @@ endif::[]
 = Accessing Prometheus and Grafana
 
 link:https://prometheus.io/[Prometheus] is an open-source systems and service monitoring and alerting toolkit that you can use to monitor services deployed in your Red Hat OpenShift environment. Prometheus collects and stores metrics from configured services at given intervals, evaluates rule expressions, displays the results, and can trigger alerts if a specified condition becomes true.
-Grafana is open source visualization and analytics software. It allows you to query, visualize, alert on, and explore your metrics.
+
+link:https://grafana.com/[Grafana] is open source visualization and analytics software. It allows you to query, visualize, alert on, and explore your metrics.
```
Comment (sjay72) [inline modules/ROOT/pages/_partials/gs-accessing-prometheus-grafana-proc.adoc]:
```suggestion
link:https://grafana.com/[Grafana] is open source visualization and analytics software. It enables you to query, visualize, set alert thresholds, and explore your metrics.
```

File: modules/ROOT/pages/_partials/gs-accessing-prometheus-grafana-proc.adoc [hunk 3]
```diff
@@ -7,15 +7,17 @@ endif::[]
...
 
 .Prerequisite
-* You are logged in to the OpenShift console as the `customer-admin` user.
+* You are logged in to the {webapp} as the `customer-admin` user.
 
 .Procedure
-. In the *My Projects* section, click *View All*.
+. On the *All Services* tab, click the *Open console* button for *Red Hat OpenShift*.
```
Comment (sjay72) [inline modules/ROOT/pages/_partials/gs-accessing-prometheus-grafana-proc.adoc]:
```suggestion
. On the *All Services* tab, locate the entry for Red Hat OpenShift and click *Open console*.
```

Comment (sjay72) [reply modules/ROOT/pages/_partials/gs-accessing-prometheus-grafana-proc.adoc]: IBM style pp 212

**Review by sjay72 — COMMENTED** [review]
Looking good, Brian.

**Pull request merged**